packages/@tinacms/cli/src/cmds/statusChecks/checkClientInformation.ts

```diff
@@ -0,0 +1,47 @@
+/**
+Copyright 2021 Forestry.io Holdings, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import Progress from 'progress'
+
+export const checkClientInfo = async (
+  ctx,
+  next,
+  _options: { verbose?: boolean }
+) => {
+  const client = ctx.client
+
+  const bar = new Progress('Checking clientId, token and branch. :prog', 1)
+
+  try {
+    await client.request({
+      query: `query {
+        collections {
+          name
+        }
+      }`,
+    })
+    bar.tick({
+      prog: '✅',
+    })
+  } catch (e) {
+    bar.tick({
+      prog: '❌',
+    })
+    console.warn(
+      'Error when checking client information. Please check you have the correct client ID, URL and read only token configured. For more information see https://tina.io/docs/tina-cloud/connecting-site/'
```

Comment:
I think it'd be helpful to list the values we have here. From my experience I usually forget the token altogether or get the branch wrong.

Reply:
Agreed.